### PR TITLE
Bochs: Make SDL2 dependency optional

### PR DESCRIPTION
**Pull request by ronmrdechai**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by ronmrdechai** [comment]
Why does the build fail? Builds fine on my machine with and without sdl2

**Comment by MikeMcQuaid** [comment]
@ronmrdechai Please click through to the CI results to see: https://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/87/version=sierra/testReport/junit/brew-test-bot/sierra/style_bochs/

Run `brew style bochs` and fix the warning, thanks.

File: Formula/bochs.rb [hunk 1]
```diff
@@ -48,6 +48,9 @@ def install
       --with-term
     ]
 
+
```
Comment (MikeMcQuaid) [inline Formula/bochs.rb]: Don't need this extra empty line

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks again for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by ronmrdechai** [comment]
Thank you!

**Pull request merged**